### PR TITLE
chore: Add Kura 5.3.0 release notes

### PR DESCRIPTION
**Pull request by nicolatimeus**
> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped.

**Screenshots:** If applicable, add screenshots to help explain your solution

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

File: kura/distrib/RELEASE_NOTES.txt [hunk 1]
```diff
@@ -1,550 +1,256 @@
...
-
-
-Compatibility:
-  * Eclipse Kura v4.0.0 introduces API breakage with previous releases.
-  * Containing a folder restructuring, a cleanup of the destination filesystem is required before installing the new version of Kura
-  * The Command Service is now disabled by default. To use it, the user needs to opt-in to this feature, enabling the service
-    from configuration.
-
+Features:
+  * 7a54e07345 - First implementation of NetworkStatusService REST APIs (#4461) (nicolatimeus)
```
Comment (marcellorinaldo) [inline kura/distrib/RELEASE_NOTES.txt:10]: ` * 5743b23 - [dhcp-server] added dnsmasq support [Backport]`

File: kura/distrib/RELEASE_NOTES.txt [hunk 2]
```diff
@@ -555,1478 +261,272 @@ Know Issues :
...
-   * Eclipse based emulator for local host PC development/testing including support for OSX and Linux
-   * Remote configuration and device management capabilities when connected to Eurotech's Everyware Cloud
-   * Support for remote OSGi console and bundle deployment from Eclipse
-
-Known Issues:
-   * Hardware watchdog: not implemented on all platforms
-   * Reliagate 50-21: Using eth1 and Windows 7 to access the webui does not work over some network topologies - OSX or Linux are recommended to access the webui in this rare case
-   * Only one WAN interface is currently supported.  Make sure you never have multiple WAN interfaces defined or it could result in network connectivity issues
+Changelog:
+  * 56800061a5 - chore: Updated libsocket-can-osgi version (#4495) (nicolatimeus)
```
Comment (marcellorinaldo) [inline kura/distrib/RELEASE_NOTES.txt:281]: ` * 5743b23 - feat(dhcp-server): added dnsmasq support [Backport]`

**Review by marcellorinaldo — COMMENTED** [review]
New PR merged.

**Review by MMaiero — APPROVED** [review]

**Review by salvatore-coppola — APPROVED** [review]

File: kura/distrib/RELEASE_NOTES.txt [hunk 3]
```diff
@@ -1,550 +1,256 @@
-Eclipse Kura - 4.1.1   - May 2020
--------------------------------------------------------------------------
+Eclipse Kura - 5.3.0 - March 2023
+-------------------------------------------------------------------------------------------------
 Description:
-Eclipse Kura 4.1.1 is a service release to address various discovered bugs and functional issues.
+This minor release of Eclipse Kura, compatible with Java 8 and OSGi R7, introduces a new framework compatibility with Java 17, a completely renewed networking that leverages Linux NetworkManager, a new set of APIs and abstarctions for SQL and noSQL databases, the availability of SQLite as a database option (H2 still available for compatibility reasons), an improved Container Management feature with UDP ports support, a refreshed Asset implementation with the new data publishing on change and, finally, the new Kura Data Service Connection Policy.
```
Comment (mattdibi) [inline kura/distrib/RELEASE_NOTES.txt]:
```suggestion
This minor release of Eclipse Kura, compatible with Java 8 and OSGi R7, introduces a new framework compatibility with Java 17, a completely renewed networking that leverages Linux NetworkManager, a new set of APIs and abstractions for SQL and noSQL databases, the availability of SQLite as a database option (H2 still available for compatibility reasons), an improved Container Management feature with UDP ports support, a refreshed Asset implementation with the new data publishing on change and, finally, the new Kura Data Service Connection Policy.
```

**Review by marcellorinaldo — APPROVED** [review]
My PRs seem to be all there

**Review by marcellorinaldo — APPROVED** [review]

**Pull request merged**